exe-common/src/config.rs

```diff
@@ -204,6 +205,7 @@ pub mod net {
             Config {
                 genesis: HeaderHash::from_hex(&"89D9B5A5B8DDC8D7E5A6795E9774D97FAF1EFEA59B2CAF7EAF9F8C5B32059DF4").unwrap(),
                 genesis_prev: HeaderHash::from_hex(&"5f20df933584822601f9e3f8c024eb5eb252fe8cefb24d1317dc3d432e940ebb").unwrap(),
+                k: 2160,
```

Comment:
at the top of this file, use a const to define this value:

```rust
const DEFAULT_EPOCH_STABILITY_DEPTH : usize = 2160;
```

also use `usize` here, so we can easily compare it with the number of hases in a collections, number of file in a directory, which are often typed as `usize`.

Reply:
The reason for using a `u32` was to match the type of `SlotId`.